diff --git a/src/sudo/mod.rs b/src/sudo/mod.rs
index 9fd25af0..3dabd907 100644
--- a/src/sudo/mod.rs
+++ b/src/sudo/mod.rs
@@ -47,11 +47,10 @@ const VERSION: &str = if let Some(version_override) = std::option_env!("SUDO_RS_
 };
 
 pub(crate) fn candidate_sudoers_file() -> PathBuf {
-    let mut path = if cfg!(target_os = "freebsd") {
-        option_env!("LOCALBASE").unwrap_or("/usr/local").into()
-    } else {
-        PathBuf::from("/")
-    };
+    let mut path = std::env::var("LOCALBASE")
+        .map(PathBuf::from)
+        .unwrap_or(PathBuf::from("/"));
+
     path.push("etc/sudoers-rs");
     if !path.exists() {
         path.set_file_name("sudoers");
